superset/databases/commands/update.py

```diff
@@ -54,7 +54,7 @@ def run(self) -> Model:
             # TODO Improve this simplistic implementation for catching DB conn fails
             try:
                 schemas = database.get_all_schema_names()
-            except Exception:
+            except Exception as ex:
```

Comment:
Should we wrap `ex` in the `DatabaseConnectionFailedError`?

Reply:
We shouldn't because all sorts of info (or even sensitive info) may come out of it